configs/opinicus_config

```diff
@@ -2661,7 +2671,7 @@ CONFIG_GPIO_WATCHDOG=m
 CONFIG_OMAP_WATCHDOG=m
 # CONFIG_DAVINCI_WATCHDOG is not set
 CONFIG_RN5T618_WATCHDOG=m
-CONFIG_SUNXI_WATCHDOG=m
+CONFIG_SUNXI_WATCHDOG=y
```

Comment:
don't compile it in, copy the module instead, as we need to do that anyway for the display bits as well